content/account/app/index.textile

```diff
@@ -0,0 +1,97 @@
+meta_keywords: “Ably dashboard, manage applications, monitor apps, app creation, application settings, Ably platform”
+---
+
+Manage and monitor your applications on the Ably platform using the Ably dashboard. Create new apps, view existing ones, and configure settings from your browser.
+
+Begin by "logging":https://ably.com/login in to Ably through your browser. Once you're logged in, you have access to the Ably "dashboard":https://ably.com/accounts, where you can "create a new app":#create or "view your existing apps.":#view
+
+h2(#create). Create a new app
+
+The following image displays the Ably dashboard when no apps have been created yet. To get started, you can easily *Create new app*:
+
+<a href="@content/screenshots/dash/create-app.png" target="_blank">
+  <img src="@content/screenshots/dash/create-app.png" style="width: 100%" alt="Create apps in the Ably dashboard">
+</a>
+
+h2(#view). View your apps
+
+If you have created apps already, your dashboard will look similar to the one shown in the following image. Each app displayed is a link to its specific dashboard, where you can view and configure settings, as well as monitor usage and statistics:
+
+<a href="@content/screenshots/dash/your-apps.png" target="_blank">
+  <img src="@content/screenshots/dash/your-apps.png" style="width: 100%" alt="Your apps in the Ably dashboard">
+</a>
+
+h2. Dashboard overview
+
+Your app dashboard provides the following tabs for monitoring and configuring your app settings:
+
+<a href="@content/screenshots/dash/dash-tabs.png" target="_blank">
+  <img src="@content/screenshots/dash/dash-tabs.png" style="width: 100%" alt="Your app dashboard tabs">
+</a>
+
+h3. Stats
+
+Monitor and analyze your app's performance through a "stats":/account/app/stats table and chart:
+
+* View last realtime data to quickly spot app issues.
+* Track messages, presence events, and performance metrics to understand usage and capacity.
+* Compare data over time to identify trends and optimize resources.
+* Adjust chart views with zoom options and set specific time ranges for detailed insights.
+
+h3. Getting started
+
+Get started by connecting to Ably, and publishing your first message.
+
+h3. API keys
+
+Create, manage, and configure "API keys":/account/dashboard/api by setting capabilities:
+
+* Users can create a new API key, assigning a descriptive name for easy identification.
+* Capabilities include permissions like publishing messages, subscribing to channels, retrieving message history, managing presence, and accessing statistics.
+* Control access by defining whether the key has unrestricted access, channel-only, queue-only, or specific access to selected channels and queues.
+* Users can set up integrations and use revocable tokens for enhanced security by managing token lifetimes and revocation abilities.
```

Comment:
I'm not sure what we're referring to here in terms of integrations.

Reply:
...removed -- I was getting ahead of myself